### PR TITLE
Upgrading Chaos to 0.6.3 (env variables for http auth and ssl)

### PR DESCRIPTION
**Pull request by swartzrock**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -65,7 +65,8 @@
     </repository>
     <repository>
       <id>libs-folder</id>
-      <url>file://${basedir}/lib</url>
+      <name>libs-folder</name>
+      <url>file:${project.basedir}/lib2</url>
```
Comment (elingg) [inline pom.xml]: The lib folder should just be named lib, no need to rename it.

**Comment by elingg** [comment]
LGTM, just one comment to be addressed.

**Comment by swartzrock** [comment]
Thanks for catching that error! Now fixed to just "lib". Awesome!

**Comment by elingg** [comment]
:+1:

**Pull request merged**